src/scrape.ts

```diff
@@ -179,7 +107,16 @@ const main = async () => {
       await fs.mkdir(path.dirname(metadataFile), { recursive: true });
       await fs.writeFile(metadataFile, JSON.stringify(webhook, null, 2), "utf-8");
 
-      const schema = inferSchemaFromExamplePayload(webhook.response, webhook, version);
+      const { schema, warnings: warningCount, errors: errorMessages } = inferSchemaFromExamplePayload(webhook.response, webhook);
```

Comment:
This file is mostly the same except with the removed code moving to `infer-schema`. This is the main change, instead of the console output happening inside `inferSchemaFromExamplePayload` it is moved here.